pkg/capacity/manager.go

```diff
@@ -134,9 +154,31 @@ func getOptimalTargetClusters(preferredCluster string, getMemberClusters cluster
 		}
 	}
 
-	memberNames := make([]string, len(members))
-	for i := range members {
-		memberNames[i] = members[i].Name
+	var memberNames []string
+	for _, member := range members {
+		// if cluster-role labels were provided, it will check for matching on the member labels
+		// if no clusterRoles labels are required, then the function will simple return true
+		if hasClusterRoles(clusterRoles, member) {
```

Comment:
if no clusterRole is provided, then we should automatically pick the `tenant` cluster. I also mentioned it in the Jira:
> Capacity manager is updated so it reflects the roles (if specified and targetCluster is empty) and selects the right member clusters. If no role nor name is provided then it uses the standard "tenant" cluster defined by the label cluster-role.toolchain.dev.openshift.com/tenant: "".

Reply:
I was assuming all member cluster have the `tenant` role, but I agree we can force it in case this will change in the future.

I've changed the `hasClusterRoles` to add the `tenant` role in case `clusterRoles` is empty, see https://github.com/codeready-toolchain/host-operator/pull/736/commits/6b01926137a7f5c783ed0b886e61eaba8cc13642